parsons/utilities/datetime.py

```diff
@@ -25,6 +25,17 @@ def date_to_timestamp(value, tzinfo=datetime.timezone.utc):
     return int(parsed_date.timestamp())
 
 
+def unix_convert(ts):
```

Comment:
can we change this to `convert_unix_to_readable` or similar? just to make the name slightly more specific and descriptive?